pkg/acsengine/defaults-apiserver.go

```diff
@@ -52,12 +45,12 @@ func setAPIServerConfig(cs *api.ContainerService) {
 
 	// Data Encryption at REST configuration
 	if helpers.IsTrueBoolPointer(o.KubernetesConfig.EnableDataEncryptionAtRest) {
-		staticLinuxAPIServerConfig["--experimental-encryption-provider-config"] = "/etc/kubernetes/encryption-config.yaml"
+		staticAPIServerConfig["--experimental-encryption-provider-config"] = "/etc/kubernetes/encryption-config.yaml"
 	}
 
 	// Data Encryption at REST with external KMS configuration
 	if helpers.IsTrueBoolPointer(o.KubernetesConfig.EnableEncryptionWithExternalKms) {
-		staticLinuxAPIServerConfig["--experimental-encryption-provider-config"] = "/etc/kubernetes/encryption-config.yaml"
+		staticAPIServerConfig["--experimental-encryption-provider-config"] = "/etc/kubernetes/encryption-config.yaml"
```

Comment:
Can you go ahead and combine these two conditions in an OR? I had it in the other PR

Reply:
Indeed, thanks! and done